src/static/templates.json

```diff
@@ -31,13 +31,7 @@
     },
     {
         "id": "ANDROID_1080P",
-        "preset": "Android 1080p (1920, 1080)",
-        "width": 1920,
-        "height": 1080
-    },
-    {
-        "id": "MS_SURFACE_PRO",
-        "preset": "Microsoft Surface Pro (1920, 1080)",
+        "preset": "Android 1080p (1080, 1920)",
         "width": 1920,
```

Comment:
No, the width and height _properties_ need to be switched, not the values in the strong. When you create an `Android 1080p (1080, 1920)` document you get a document of width 1080 and height 1920, but this file has `width: 1920` and `height: 1080`. Those values are used here:
![android](https://s3.amazonaws.com/f.cl.ly/items/221T1Y3N323P3C2B0L3C/Image%202015-08-04%20at%205.41.26%20PM.png)